co2eq/energy/index.js

```diff
@@ -0,0 +1,79 @@
+import md5 from 'tiny-hashes/md5';
+
+import {
+  ELECTRICITY_ACTIVITIES,
+} from '../../definitions';
+import energyFootprints from './energyfootprints.yml';
+
+// ** modelName must not be changed. If changed then old activities will not be re-calculated **
+export const modelName = 'energy';
+const ymlChecksum = md5([].map(JSON.stringify).reduce((a, b) => a + b, ''));
+export const modelVersion = `1_${ymlChecksum}`; // This model relies on footprints.yaml
```

Comment:
`ymlChecksum` needs energyFootprints as input, right?

Reply:
you're absolutely right